stored_requests/fetcher.go

```diff
@@ -191,6 +193,28 @@ func (f *fetcherWithCache) FetchRequests(ctx context.Context, requestIDs []strin
 	return
 }
 
+func (f *fetcherWithCache) FetchResponses(ctx context.Context, ids []string) (data map[string]json.RawMessage, errs []error) {
+	data = f.cache.Responses.Get(ctx, ids)
+
+	leftoverResp := findLeftovers(ids, data)
+
+	// Record cache hits for stored responses
+	//f.metricsEngine.RecordStoredReqCacheResult(metrics.CacheHit, len(ids)-len(leftoverResp))
+	// Record cache misses for stored responses
+	//f.metricsEngine.RecordStoredImpCacheResult(metrics.CacheMiss, len(leftoverResp))
```

Comment:
Is this commented out code still needed?

Reply:
Removed